RPIPs/RPIP-bounty_standards.md

```diff
@@ -0,0 +1,101 @@
+**Draft - Rejected** - Bounty was rejected by the GMC at this time. Bounty is not eligible for completion.  
+**Open** - Bounty is adopted by the GMC for completion and listed on relevant frontends. Bounty is now eligible for completion.  
+**Closed - Completed** - A Bounty has been fully completed and paid out by the GMC. Bounty is no longer eligible for completion.  
+**Closed - Expired** - Bounty has met its expiration date. Bounty is no longer eligible for completion.  
+**Closed - Withdrawn** - GMC has withdrawn the bounty. Bounty is no longer eligible for completion.  
+
+### Bounty Incentive Levers
+Bounty Incentives Levers are OPTIONAL incentives that the pDAO empowers the GMC to use to incentivise the generation and successful completion of bounty proposals.
+
+#### Lever Incentives
+* The incentive attached to each lever MAY be set by the GMC. 
+* Levers MAY have an absolute (fixed amount) component and/or a relative (percentage of total bounty payout) component. 
+* The payout of incentives for a given bounty MUST NOT reduce the bounty completion payout. 
+* When incentive levers are modified by the GMC, incentives SHOULD be paid out in such a way that ambiguity favours incentive recipients.
+* The total cost of lever incentives is not fixed. The GMC SHOULD set lever incentives with caution when funds are limited. 
+
+#### Payout
+* Bounty incentives SHOULD be paid out of the GMC budget.
+* Incentive payouts SHOULD take place within the GMC's regular payout schedule.
+* Incentive payouts MUST be denominated in dollars, and MUST be paid out in one of the GMC's accepted payout token options selected by the recipient.
+
+#### Lever A: Proposal Incentive
+The proposal incentive lever aims to influence the number of bounty proposals that are recieved by the GMC.
+* This incentive MUST be paid to the proposal author for a bounty proposal that is adopted by the GMC. 
+* Individual incentive payouts for this lever MAY be reduced at the GMC's discretion based on proposal quality and completeness.
+* Bounty definitions do not need to be included for a proposer to recieve this incentive.
+
+#### Lever B: Definition Incentive
+The definition incentive lever aims to influence the quality and completeness of bounty definitions accompanying bounty proposals.
+* This incentive MUST be paid to the definiton author for bounty definitions provided alongside a bounty proposal that is adopted by the GMC. 
+* This incentive payout SHOULD be paid a maximum once per proposal, even if the bounty proposal is split into multiple definitions.
+* Individual incentive payouts for this lever MAY be reduced at the GMC's discretion based on definition quality and completeness.
+* This incentive SHOULD go to the proposal author unless an alternative is explicitly preapproved by the GMC adminitrator.
+
+#### Lever C: Completion Incentive
```

Comment:
Do Completion Incentive Levers replace bounty awards? Or are they an additional payment on top of the bounty award by adding stricter requirements?

Reply:
I think I'll rename it to support incentive. It's giving the wrong impression.